packages/shared/lib/errors.ts

```diff
@@ -16,25 +16,8 @@ You should have received a copy of the GNU Lesser General Public License
 along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-const tryGetString = (obj: any): string | null => (typeof obj.toString === 'function' ? obj.toString() : null)
-
-export interface APIError {
-  error: {
-    detail?: string
-    message?: string
-  }
-  status: number
-  statusText: string
-}
-
-const isHTTPError = (e: unknown): e is APIError => (e as APIError).error !== undefined
+export const getHumanReadableError = (e: unknown, defaultErrorMsg: string) => tryGetString(e) || defaultErrorMsg
 
-export const getHumanReadableError = (e: unknown, defaultErrorMessage: string) => {
-  const stringifiedError = tryGetString(e)
-  const serverError = isHTTPError(e)
-    ? e.error.detail || `(${e.status}${e.statusText && `: ${e.statusText}`}) ${e.error.message || ''}`
-    : stringifiedError
-
-  return `${defaultErrorMessage}${serverError && `: ${serverError}`}`
-}
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const tryGetString = (obj: any): string | null =>
+  typeof obj.toString === 'function' ? obj.toString().replace('Error: [API Error] - ', '') : null
```

Comment:
I thought I'd clean up the error message before displaying to the user, WDYT?

Reply:
| Before | After |
| ----- | ----- |
| <img width="1312" alt="image" src="https://github.com/alephium/alephium-frontend/assets/1579899/85b46a86-5cd3-4faf-b73c-730bdc99e242"> | <img width="1312" alt="image" src="https://github.com/alephium/alephium-frontend/assets/1579899/5bf1e0de-0d3b-457e-ab14-8c3b41df235c"> |

Reply:
Nice, but what about naming the function `tryGetErrorString`?

Reply:
I simplified it even further.

Reply:
Ready for another review :)